src/components/BottomBar.js

```diff
@@ -52,6 +56,8 @@ function BottomBar ({programStarted, setProgramStarted}) {
     window.CrComLib.subscribeState('b', '95', value => setShowVolumeModal(value));
     window.CrComLib.subscribeState('b', '97', value => setShowMicModal(value));
     window.CrComLib.subscribeState('b', '88', value => setShowCamModal(value));
+    window.CrComLib.subscribeState('b', '79', value=> setConfigPrivacyMode(value));
```

Comment:
can we move away from 'magic' numbers here? Consider creating an `enum` that has semantic names, for instance: 
```js
const CrSignalType = {
  'IDLE': 'b',
  'RUNNING': 'c',
  ...
};

const CrSignalName = {
  'IDLE': '95',
  'RUNNING' :'96',
  ...
};
```

Then, in the code, use: 
```js
window.CrComLib.subscribeState(CrSignalType.IDLE, CrSignalName.IDLE, value=> setConfigPrivacyMode(value));
```

Both the CrSignalType and CrSignalName could be kept in their own module and imported where needed. 
Please note that these values are made up for illustration purposes only.